src/implementations/twig/components/rating-field/rating-field.html.twig

```diff
@@ -106,9 +108,10 @@
   {% if _items is not empty and _items is iterable %}
   <div class="ecl-rating-field">
     {%- for _item in _items|reverse %}
+      {% set _item_id = _item.id is not empty ? _item.id : _id ~ '-' ~ _item.value %}
```

Comment:
i think we should add an escape for the value here. `item.value|e('html_attr')`, the value is a string but we don't know what it contains..